tests/__snapshots__/mockConverter.test.ts.snap

```diff
@@ -0,0 +1,27 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Dictionary type should generate mocks for a "dictionary" type array 1`] = `
+"/* eslint-disable @typescript-eslint/no-use-before-define */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import {ComplexDto, MainDto, Role} from './pathToTypes';
+
+export const aComplexDtoAPI = (overrides?: Partial<ComplexDto>): ComplexDto => {
+  return {
+    name: 'name-complexdto',
+  ...overrides,
+  };
+};
+
+export const aMainDtoAPI = (overrides?: Partial<MainDto>): MainDto => {
+  return {
+    contributors: { 
+\\"role1\\": [],
+\\"role2\\": [],
+\\"role3\\": [],
```

Comment:
It would have been nice to create an array with one item of the type it could include. But not big deal if that's complex

Like:
```
"role1": [aComplexDto()],
```

Reply:
OK I take a note of it, that's a good idea. I suggest to move forward with it and enhance later on.
It would be great to support array of non complex types too (string, bool...).